extensions/ecc/circuit/src/weierstrass_extension.rs

```diff
@@ -49,6 +53,13 @@ pub static SECP256K1_CONFIG: Lazy<CurveConfig> = Lazy::new(|| CurveConfig {
     b: BigUint::from_u8(7u8).unwrap(),
 });
 
+pub static P256_CONFIG: Lazy<CurveConfig> = Lazy::new(|| CurveConfig {
```

Comment:
huh actually its a little weird these were here instead of in their modules. if it's minimal refactor, move into `p256`

Reply:
(actually where are we using either of these configs at all?)